### DISCUSSION TITLE
전용 유물 중 `폭주하는 마법 도구`와 `싹이 난 가지` 밸런스

### DISCUSSION THREAD

**Discussion by tykr0001** [scarf005/Marisa#126] [Balance]
2줄요약
`폭주하는 마법 도구` &rarr; 똥유물 &rarr; 상향필요
`싹이 난 가지` &rarr; 갓유물 &rarr; 하향필요

---
`폭주하는 마법 도구`:  

중독은 방어도로 막을 수 없기 때문에 매전투를 첫턴에 끝내는게 아닌 이상 무조건 데미지를 받는다. 손상 약화 취약은 한 턴도 아니고 두 턴. 특히 손상과 취약은 2막~3막 어디서든 굉장히 위협적인 디버프인데 상대적으로 덱이 정제되지 않아 방어가 약한 구간에서 다른 안정적인 에너지 유물(예를 들어 저주받은 열쇠, 핵융합 망치 등)을 쓰지 않고 이런 위험을 감수할 이유가 없다고 생각함. 개인적으로는 룬반구 이상의 리스크를 짊어지고 가는 느낌.

- 제안 사항: 중독을 3&rarr;2로, 손상 약화 취약을 2턴 &rarr; 1턴으로 조정하거나 힘과 민첩같은 이로운 버프도 pool에 추가
---
`싹이 난 가지`:  

가지를 대체하는 전용 유물이라지만 유물 하나의 밸류가 말도 안됨. 피를 담은 병, 불타오르는 혈액 등의 완벽한 상위호환. 배를 째고 원하는대로 덱을 빌딩할 수 있게 해주는 유지력을 제공함.  

- 제안 사항: 재생을 5 &rarr; 4로 조정
---

`요술 빗자루`도 사실 별이 담긴 상자, 가속 등 0코 관련 카드 및 부채와의 연계가 너무 좋아서 대부분의 경우 무한덱이 완성되니 문제가 있지 않나.. 라고 생각하는데 상대적으로 약한 스파크 키워드를 생각하면 또 냅둬도 될 것 같기도 하고 해서 언급만 해봤음

**Comment by scarf005** [comment]
의견 ㄱㅅㄱㅅ
#128 에서 추적중
